src/shared/canvas/MyData/types.ts

```diff
@@ -7,7 +7,7 @@ export type TFilterOptions = {
   query: string;
   dateFilterType?: TDateFilterType;
   dateField: TDateField;
-  dataType?: string[];
+  types?: TType[];
```

Comment:
If we include the first change I suggest, we can remove the `?` from type definiton here too.